content/de/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -0,0 +1,119 @@
+linkTitle: Kubernetes Grundlagen lernen
+weight: 10
+card:
+  name: tutorials
+  weight: 20
+  title: Schritt-für-Schritt Grundlagen
+---
+
+<!DOCTYPE html>
+
+<html lang="de">
+
+<body>
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>Kubernetes Grundlagen</h2>
+        <p>Dieses Tutorial bietet einen Überblick über die Grundlagen des Kubernetes-Cluster-Orchestrierungssystems. Jedes Modul enthält einige Hintergrundinformationen zu den wichtigsten Funktionen und Konzepten von Kubernetes sowie ein interaktives Online-Lernprogramm. Mit diesen interaktiven Lernprogrammen können Sie einen einfachen Cluster und seine containerisierten Anwendungen selbst verwalten.</p>
+        <p>Mithilfe der interaktiven Tutorials können Sie Folgendes lernen:</p>
+        <ul>
+        <li>Eine containerisierte Anwendung in einem Cluster bereitstellen</li>
+        <li>Skalieren des Deployments</li>
+        <li>Aktualisieren der containerisierten Anwendung mit einer neuen Softwareversion</li>
+        <li>Debuggen der containerisierten Anwendung</li>
+        </ul>
+        <p>In den Lernprogrammen wird mit Katacoda ein virtuelles Terminal in Ihrem Webbrowser ausgeführt, in dem Minikube ausgeführt wird. Dies ist eine kleine lokale Bereitstellung von Kubernetes, die überall ausgeführt werden kann. Es muss keine Software installiert oder konfiguriert werden. Jedes interaktive Lernprogramm wird direkt von Ihrem Webbrowser aus ausgeführt.</p>
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>Was kann Kubernetes für Sie tun?</h2>
+        <p>Bei modernen Webservices erwarten Benutzer, dass Anwendungen rund um die Uhr verfügbar sind, und Entwickler erwarten, mehrmals täglich neue Versionen dieser Anwendungen bereitzustellen (deployen). 
+          Containerisierung hilft bei der Paketierung von Software, um diese Ziele zu erreichen, sodass Anwendungen einfach und schnell ohne Ausfallzeiten veröffentlicht und aktualisiert werden können. Kubernetes hilft Ihnen dabei, sicherzustellen, dass diese Containeranwendungen immer dort laufen, wo und wann Sie möchten, und hilft ihnen, die Ressourcen und Tools zu finden, die sie zum Arbeiten benötigen. Kubernetes ist eine produktionsbereite Open-Source-Plattform, die auf der gesammelten Erfahrung von Google in der Container-Orchestrierung basiert und mit den besten Ideen der Community kombiniert wird.</p>
+      </div>
+    </div>
+
+    <div id="basics-modules" class="content__modules">
+      <h2>Kubernets Grundlagen Module</h2>
+      <div class="row">
+        <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/de/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
```

Comment:
It is better to remove the `/de` part. We are looking into a way of adding the language code automatically ;)

```suggestion
                <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
```